README.md

```diff
@@ -0,0 +1,467 @@
+# VGM Tools 
+by Valley Bell (@valleybell)
+
+## Tool List
+[dro2vgm - DRO to VGM Converter](#dro-to-vgm-converter-dro2vgm)  
+[imf2vgm - IMF to VGM Converter](#imf-to-vgm-converter-imf2vgm)  
+[opl_23 - OPL 2<->3 Converter](#opl-2-3-converter-opl_23)  
+[opt_oki - VGM OKIM6258 Optimizer](#vgm-okim6258-optimizer-opt_oki)  
+[optdac - VGM DAC Optimizer](#vgm-dac-optimizer-optdac)  
+[optvgm32 - VGM PWM Optimizer](#vgm-pwm-optimizer-optvgm32)  
+[optvgmrf - VGM RF-PCM Optimizer](#vgm-rf-pcm-optimizer-optvgmrf)  
+[raw2vgm - RAW to VGM Converter](#raw-to-vgm-converter-raw2vgm)  
+[vgm_cmp - VGM Compressor](#vgm-compressor-vgm_cmp)  
+[vgm_cnt - VGM Command Counter](#vgm-command-counter-vgm_cnt)  
+[vgm_dbc - VGM Data Block Compressor](#vgm-data-block-compressor-vgm_dbc)  
+[vgm_dso - VGM DAC Stream Optimizer](#vgm-dac-stream-optimizer-vgm_dso)  
+[vgm_facc - Make VGM Frame Accurate](#make-vgm-frame-accurate-vgm_facc)  
+[vgm_mono - VGM Mono](#vgm-mono-vgm_mono)  
+[vgm_ndlz - VGM Undualizer](#vgm-undualizer-vgm_ndlz)  
+[vgm_ptch - VGM Patcher](#vgm-patcher-vgm_ptch)  
+[vgm_smp1 - Remove 1 Sample Delays](#remove-1-sample-delays-vgm_smp1)  
+[vgm_sptd - VGM Splitter (Delay Edition)](#vgm-splitter-delay-edition-vgm_sptd)  
+[vgm_spts - VGM Splitter (Sample Edition)](#vgm-splitter-sample-edition-vgm_spts)  
+[vgm_sro - VGM Sample-ROM Optimizer](#vgm-sample-rom-optimizer-vgm_sro)  
+[vgm_stat - VGM Statistics](#vgm-statistics-vgm_stat)  
+[vgm_tag - VGM Tagger](#vgm-tagger-vgm_tag)  
+[vgm_tt - VGM Tag Transfer](#vgm-tag-transfer-vgm_tt)  
+[vgm_trim - VGM Trimmer](#vgm-trimmer-vgm_trim)  
+[vgm_vol - VGM Volume Detector](#vgm-volume-detector-vgm_vol)  
+[vgm2txt - VGM Text Writer](#vgm-text-writer-vgm2txt)  
+[vgmlpfnd - VGM Loop Finder](#vgm-loop-finder-vgmlpfnd)  
+[vgmmerge - VGM Merger](#vgm-merger-vgmmerge)  
+
+### DRO to VGM Converter (dro2vgm)
+---
```

Comment:
I think you don't really need the horizontal lines here.
The text file needed them to indicate head lines, but in markdown you already do that with ###.

(Yes, I'm nitpicking.)

Reply:
In GitHub Markdown, Header Lines don't appear for H2-4, only H1. See example:

https://github.com/birdybro/vgmtools/blob/header-lines-check/README.md#imf-to-vgm-converter-imf2vgm

I did the lines on purpose as I noticed this partway through as I was removing them :)